From ee66ee892c2f7a965d78f1e5c8135b2f761f6e21 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 31 Aug 2023 08:53:21 +0000
Subject: [PATCH] Enable omx service although hal is deprecated

OMX service is needed even if hal is deprecated
Since decoder depends on omx apis.

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/product.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/groups/device-specific/caas/product.mk b/groups/device-specific/caas/product.mk
index 434eda5..a1d97e9 100755
--- a/groups/device-specific/caas/product.mk
+++ b/groups/device-specific/caas/product.mk
@@ -32,6 +32,7 @@ PRODUCT_PACKAGES +=  \
                     android.hardware.identity \
                     android.hardware.identity-service.example \
                     android.hardware.graphics.composer3-service.intel \
+		    android.hardware.media.omx@1.0-service
 
 
 PRODUCT_PROPERTY_OVERRIDES += ro.control_privapp_permissions=enforce
-- 
2.39.2

